clouds/clouds-aws/src/test/resources/aws/cloudformation/eap.yaml

```diff
@@ -0,0 +1,186 @@
+        Properties:
+            Tags:
+                -   Key: "cf-template"
+                    Value: !Ref defaultTag
+            CidrBlock: "172.31.0.0/16"
+            EnableDnsSupport: true
+            EnableDnsHostnames: true
+            InstanceTenancy: "default"
+
+    EC2SecurityGroup:
+        Type: "AWS::EC2::SecurityGroup"
+        Properties:
+            Tags:
+                -   Key: "cf-template"
+                    Value: !Ref defaultTag
+            GroupDescription: "Allows any connection"
+            GroupName: "allow-any"
+            VpcId: !Ref EC2VPC
+            SecurityGroupIngress:
+                -   CidrIp: "0.0.0.0/0"
+                    IpProtocol: "-1"
+                -   CidrIpv6: "::/0"
+                    IpProtocol: "-1"
+            SecurityGroupEgress:
+                -   CidrIp: "0.0.0.0/0"
+                    IpProtocol: "-1"
+
+    EC2InstanceEAP:
+        Type: "AWS::EC2::Instance"
+        CreationPolicy:
+            ResourceSignal:
+                Timeout: PT5M
+        DependsOn:
+            - EC2KeyPair
+        Properties:
+            ImageId: !Ref ImageID
+            InstanceType: "t3.medium"
+            KeyName: !Ref EC2KeyPair
+            AvailabilityZone: !GetAtt EC2Subnet.AvailabilityZone
+            Tenancy: "default"
+            SubnetId: !Ref EC2Subnet
+            EbsOptimized: false
+            SecurityGroupIds:
+                - !Ref EC2SecurityGroup
+            SourceDestCheck: true
+            IamInstanceProfile: EC2InstanceRole
+            BlockDeviceMappings:
+                -   DeviceName: "/dev/sda1"
+                    Ebs:
+                        Encrypted: false
+                        VolumeSize: 10
+                        VolumeType: "gp2"
+                        DeleteOnTermination: true
+            Tags:
+                -   Key: "cf-template"
+                    Value: !Ref defaultTag
+                -   Key: "cf-template-resource"
+                    Value: !Join [ "-", [!Ref defaultTag, "ec2instance"]]
+                -   Key: "Name"
+                    Value: !Ref instanceName
+            HibernationOptions:
+                Configured: false
+            CpuOptions:
+                CoreCount: 1
+                ThreadsPerCore: 2
+            EnclaveOptions:
+                Enabled: false
+            UserData:
+                Fn::Base64:
+                    !Sub |
+                        #!/usr/bin/env bash
+                        JBOSS_HOME=/opt/rh/eap7/root/usr/share/wildfly
+
+                        yum install -y unzip python3-pip
+                        pip3 install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz
+                        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+                        unzip -q awscliv2.zip
+                        chmod +x ./aws/dist/aws
+                        export PATH=$PATH:`pwd`/aws/dist
```

Comment:
is this used in the end?

Reply:
Oh, this may be some leftover from some other complicated template. I will remove it.

Reply:
Fixed.